controller/src/database/mod.rs

```diff
@@ -19,15 +31,17 @@ impl RikDataBase {
     pub fn init_tables(&self) -> Result<()> {
```

Comment:
Is the method using this method handling this error ?

The idea is not to lift the error without having a proper error handling in the upper levels (could be as simple as a logging, or using an alternative solution)